app/assets/javascripts/shared/inline_editing.js

```diff
@@ -180,15 +180,14 @@ var inlineEditing = (function() {
         .removeClass('hidden');
       e.stopPropagation();
     })
-    .on('keyup', `${editBlocks} input`, function(e) {
+    .on('keydown', `${editBlocks} input`, function(e) {
       var container = $(this).closest(editBlocks);
       if (e.keyCode === 13) {
         updateField(container);
+      } else if (e.keyCode === 9) {
```

Comment:
I think we should set some constants to these keycodes.
It will save ups some time looking them up every time.